guides/common/modules/ref_pxe-loaders.adoc

```diff
@@ -0,0 +1,84 @@
+:_mod-docs-content-type: REFERENCE
+
+[id="pxe-loaders"]
+= PXE loaders
+
+In {Project}, a PXE loader defines the `filename` DHCPv4 option to use during provisioning.
+{Project} deploys the required files to TFTP {SmartProxy}.
+
+.Limitations
+* {Project} can deploy the required booting files only for the `x86_64` architecture.
+ifdef::satellite[]
+* {Team} does not provide support for iPXE workflows.
+endif::[]
+
+.PXE loaders and DHCPv4 filenames
+|===
+| PXE loader option | `filename` value | Use for
+
+| *None*
+| No filename passed
+| Non-PXE booting, for example HTTP booting with iPXE or image-based provisioning
+
+| *PXELinux BIOS*
+| `pxelinux.0`
+| BIOS machines with PXE booting
+
+| *PXELinux UEFI*
+| `pxelinux.efi`
+| UEFI machines with PXE booting
+
+| *Grub2 BIOS*
+| `grub2/grub__arch__.0`
+| BIOS machines with PXE booting
+
+| *Grub2 UEFI*
+| `__subdir__/grub2/grub__arch__.efi`
+| UEFI machines with PXE booting
+
+| *Grub2 UEFI SecureBoot*
+| `__subdir__/grub2/shim__arch__.efi`
+| SecureBoot-enabled UEFI machines with PXE booting
+
+| *Grub2 UEFI HTTP*
+| `http://__httpboot.example.com__/httpboot/__subdir__/grub2/grub__arch__.efi`
+| UEFI machines with HTTP booting
+
+| *Grub2 UEFI HTTPS*
+| `https://__httpboot.example.com__/httpboot/__subdir__/grub2/grub__arch__.efi`
+| UEFI machines with HTTP booting over HTTPS
+
+ifndef::satellite[]
+| *Grub2 UEFI HTTPS SecureBoot*
+| `https://__httpboot.example.com__/httpboot/__subdir__/grub2/shim__arch__.efi`
+| SecureBoot-enabled UEFI machines with HTTP booting over HTTPS
+endif::[]
+
+| *iPXE Embedded*
+| {Project} iPXE URL
+| ? with iPXE booting
+
+| *iPXE UEFI HTTP*
+| `http://__httpboot.example.com__/httpboot/ipxe-__arch__.efi`
+| UEFI machines with HTTP boot using iPXE implementation
+
+| *iPXE Chain BIOS*
+| `undionly-ipxe.0`
+| BIOS machines with iPXE booting
+
+| *iPXE Chain UEFI*
+| `ipxe.efi`
+| UEFI machines with iPXE booting
+|===
+
+ifndef::orcharhino,satellite[]
+Other PXE loaders, such as *PXELinux UEFI* or *Grub2 ELF*, require additional configuration.
```

Comment:
I wondered what `Grub2 ELF` actually was. https://projects.theforeman.org/issues/26051 mentions it's mainly for IBM POWER. Also found https://www.theforeman.org/2016/09/new-post.html and https://community.theforeman.org/t/netbooting-ibm-power9-via-petitboot/17104 via that. Interesting.